### ISSUE TITLE
Installed PHP_CodeSniffer globally but it does not work

### ISSUE THREAD

**Issue by jimmyko** [ikappas/vscode-phpcs#6]
The PHP and PEAR on Mac is installed via homebrew-php, then I use PEAR to install PHP_CodeSniffer. Unfortunatelly it still cannot be located.

**Comment by ikappas** [comment]
Thanks for your input. Is phpcs accessible anywhere in your terminal? ie. which phpcs reports the path of phpcs ?  

Please update to 0.5.0 and let me know if your problem persits

**Comment by jimmyko** [comment]
@ikappas
Thanks for your kind response. I just notice that the phpcs is not accessible globally. I reinstalled the phpcs with homebrew. Your plugin is working very well. Thanks for your great affort.

**Comment by jimmyko** [comment]
Please help to close this issue. Cheers. :)

**Comment by jimmyko** [comment]
@ikappas 

I just try to use composer rather than the pear to install phpcs globally, it does not work again. The path of phpcs is
```shell
which phpcs
/Users/jimmyko/.composer/vendor/bin/phpcs
```

**Comment by ikappas** [comment]
Nice to hear that it works for you. I am closing this issue.

**Comment by williamdunstanmorris** [comment]
I also tried to install PHP_CodeSniffer globally using Composer,  with  
```
composer global require "squizlabs/php_codesniffer=*"
```
and after adding 
```
~/.composer/vendor/bin/
```
to my $PATH, phpcs is still not found. I fear this may be to do with .zsh shell, 

I also tried using composer exec <command> [args...] as a way to access phpcs without appending the $PATH, but this does not seem to be working as it should. 

Anyone finding this just as difficult as me? Ive spent days on and off trying to battle this.

Any ideas?
